tiny_dnn/util/serialization_functions.h

```diff
@@ -242,6 +242,90 @@ struct LoadAndConstruct<tiny_dnn::slice_layer> {
   }
 };
 
+template <>
+struct LoadAndConstruct<tiny_dnn::sigmoid_layer> {
+  template <class Archive>
+  static void load_and_construct(
```

Comment:
@karandesai-96 @beru could we reuse code with templated parameters here and below files?